src/store/domains/ui.ts

```diff
@@ -2,7 +2,7 @@
 import { makeAutoObservable } from 'mobx';
 
 export default class UI implements UIStoreInterface {
-  lang = 'de'
+  lang:string = 'de'
```

Comment:
In Typescript kann man Type-Informationen explizit setzen, muss es aber in bestimmten Fälle nicht immer.

In diesem Falle wäre es nicht notwendig, da Typescript Typeinferenz "beherrscht". Der Typ wird dann anhand des Werts abgeleitet, der initial der Variable zugewiesen wird.